log_request_id/middleware.py

```diff
@@ -21,18 +21,21 @@ def process_request(self, request):
         request.id = request_id
 
     def get_log_message(self, request, response):
+        message = 'method=%s path=%s status=%s' % (request.method, request.path, response.status_code)
+
         user = getattr(request, 'user', None)
```

Comment:
Your test setup is missing the session middleware. Otherwise, this line would have accessed the user even for an empty session or the setting being set to None.

Reply:
Actually, scratch stat, since you're not accessing any user attribute this should be fine. However, I'd suggest rewriting this a little to make the logic easier to read. What do you think?

Reply:
Good point. I put session check above the user check.